src/lib.rs

```diff
@@ -16,13 +16,34 @@ impl zed::Extension for ScalaExtension {
             .which("metals")
             .ok_or_else(|| "Metals must be installed via coursier. Please install coursier (https://get-coursier.io/), and then run `cs install metals`.".to_string())?;
 
+        let arguments = LspSettings::for_worktree("metals", worktree)
+            .map(|lsp_settings| {
+                lsp_settings
+                    .binary
+                    .and_then(|binary| binary.arguments)
+                    // If no arguments are provided, default to enabling the HTTP server.
```

Comment:
If we default to the `-Dmetals.http=on` binary argument, should we also add `"isHttpEnabled": true` to the default LSP initialization options? :slightly_smiling_face:

Reply:
@mprihoda I have tested it on the latest Metals snapshot and had to manually set the `isHttpEnabled` initialization option in my Zed settings for the HTTP server to be working. :+1:

Reply:
I guess I will make another PR to include the isHttpEnabled as default, makes sense.

I have to admit I spent much more time trying to figure out what is the relation between the server props and initialization options than making the change to have Metals configurable - both from the documentation and from a quick glance at the Metals source code it seemed to me that `-Dmetals.http=on` and `isHttpEnabled: true` should be equivalent, the documentation even mentions that we should prefer initialization options to server props, but the behaviour seems to be off. Also, it seems the server starts after the metals is completely initialized, which includes initial build, so it might take a while before it is accessible.

I ran out of time and since I had the code for the Zed plugin ready, it seemed to me it would be better to make the PR and investigate this specific aspect later.